main/src/ca/uwaterloo/flix/language/errors/WeederError.scala

```diff
@@ -464,4 +464,43 @@ object WeederError {
     }
   }
 
+  /**
+    * An error raised to indicate type params where some (but not all) are explicitly kinded.
+    *
+    * @param loc the location where the error occurred.
+    */
+  case class InconsistentTypeParameters(loc: SourceLocation) extends WeederError {
+    def summary: String = "Inconsistent type parameters."
```

Comment:
I think we need more information here. Remember VSCode only shows the summary.

Reply:
Would it suffice just to put the tip as the summary? (`Either all or none of the type parameters must be annotated with a kind.`)

Reply:
I think that would be OK.